src/main/java/net/minecraftforge/fmp/microblock/BlockMicroMaterial.java

```diff
@@ -0,0 +1,179 @@
+package net.minecraftforge.fmp.microblock;
+
+import java.util.Map.Entry;
+
+import com.google.common.base.Function;
+import com.google.common.base.Joiner;
+import com.google.common.collect.Iterables;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.SoundType;
+import net.minecraft.block.properties.IProperty;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.BlockRenderLayer;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.Tuple;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.IBlockAccess;
+import net.minecraftforge.fml.relauncher.ReflectionHelper;
+
+/**
+ * A simple implementation of {@link IMicroMaterial} that's defined based on an {@link IBlockState}.
+ *
+ * @see IMicroMaterial
+ * @see MicroblockRegistry
+ */
+public class BlockMicroMaterial implements IMicroMaterial
```

Comment:
It's an IMicroMaterial which inherits the properties of the IBlockState that you pass in. Basically a simple block -> microblock material wrapper.